Cython/Includes/cpython/time.pxd

```diff
@@ -0,0 +1,39 @@
+"""
+Cython implementation of (parts of) the standard library time module.
+"""
+
+from libc.stdint cimport int64_t
+
+ctypedef int64_t _PyTime_t
+
+cdef extern from "pytime.h":
+    _PyTime_t _PyTime_GetSystemClock() nogil
+    double _PyTime_AsSecondsDouble(_PyTime_t t) nogil
+
+from libc.time cimport (
+    tm,
+    time_t,
+    localtime as libc_localtime,
+)
+
+
+cpdef inline double time() nogil:
```

Comment:
I think these functions should just be `cdef`. There use is to get cimported and called from Cython code, so there is no need for a Python wrapper. Besides, I actually wonder what `cpdef` does at all inside of a `.pxd` file…